cypress/integration/HeaderBar/The_HeaderBar_should_display_the_title_provided_by_the_backend_and_the_app/The_HeaderBar_displays_the_custom_title.js

```diff
@@ -4,12 +4,12 @@ import { Then, Given } from 'cypress-cucumber-preprocessor/steps'
 Given(
     'the custom title is {string} and the app title is "Example!"',
     applicationTitle => {
-        cy.fixture('HeaderBar/applicationTitle')
-            .then(response => ({
-                ...response,
-                applicationTitle,
-            }))
-            .as('applicationTitleFixture')
+        cy.get('@applicationTitleFixture').then(fx => {
+            cy.route({
+                url: 'https://domain.tld/api/system/info',
+                response: { ...fx, applicationTitle },
+            }).as('modules')
+        })
```

Comment:
Nice, this is cleaner

The `applicationTitle` fixture actually only contains the `applicationTitle` key, should we skip the `cy.get()` and just set up the route here?

Also, `.as('modules')` should be `.as('applicationTitle')` (was also wrong on `alpha`)

Reply:
> should we skip the cy.get() and just set up the route here?

It's not a costly operation. I guess if ever something is added, this part of the code doesn't have to worry about it, so I'd prefer to keep it

Reply:
> Also, .as('modules') should be .as('applicationTitle') (was also wrong on alpha)

Fixed